src/js/app.js

```
@@ -1099,6 +1146,33 @@ function openModal(idx){
+        'very-active': '🟢 Very Active',
+        'active':      '🔵 Active',
+        'moderate':    '🟡 Moderate',
+        'low':         '🔴 Low Activity',
+      };
+      mCommunity.style.display = 'block';
+      mCommunity.innerHTML = `
+        <div class="ms-title" style="margin-top:16px">Community Activity</div>
+        <div class="ca-grid">
+          <div class="ca-row"><span class="ca-label">Overall Score</span><span class="ca-value" style="font-weight:700;color:var(--orange)">${cd.score} / 100</span></div>
+          <div class="ca-row"><span class="ca-label">Tier</span><span class="ca-value">${tierMap[cd.tier] || cd.tier}</span></div>
+          <div class="ca-row"><span class="ca-label">Avg Issue Response</span><span class="ca-value">${cd.issueResponseHours != null ? cd.issueResponseHours + ' hrs' : '—'}</span></div>
```

ESLint

[error] Expected '!==' and instead saw '!='.

(eqeqeq)